src/app/modules/project/modal-highlight/modal-highlight.component.html

```diff
@@ -37,6 +37,12 @@ <h3>End Date</h3>
             <input type="date" formControlName="endDate" [disabled]="dateFieldsEnabled" />
           </div>
         </div>
+        <div class="col-12">
+          <div>
+            <h3>Project description</h3>
+            <input type="text" placeholder="Project description" formControlName="description">
+          </div>
+        </div>
         <div class="col-12" *ngIf="dateErrorMessage != null">
           <p class="date-error-message">{{dateErrorMessage}}</p>
```

Comment:
Since we're now using this "date error message" for more than just the date, it's best to rename it to something like "validation error message". 
When doing this, make sure to also rename the .scss class.